core/templates/paged_array.h

```diff
@@ -53,7 +53,12 @@ class PagedArrayPool {
 	SpinLock spin_lock;
 
 public:
-	uint32_t alloc_page() {
+	struct PageInfo {
+		T *page;
```

Comment:
I'd suggest adding initializers. That's the trend in Godot lately. I think not adding them should be the exception (for spots where perf is critical and the compiler can't be trusted to optimize out redundant assignments).

Reply:
Sounds good 👍🏻